<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>492</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Menu and toolbar</string>
   </property>
   <widget class="QRadioButton" name="uiMenuAndToolbar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Menu and toolbar</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="uiMenuOnly">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Only menu</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="uiToolbarOnly">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Only toolbar</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="statusBarEnable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>521</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Show statusbar</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enableLoadLast">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>521</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Load last file on start</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>541</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Book Design</string>
   </property>
   <widget class="QPushButton" name="bgColorButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bgColorLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Background color</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>541</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Custom CSS</string>
   </property>
   <widget class="QCheckBox" name="customCssEnable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable custom (additional) CSS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="customCssSelect">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>.</normaloff>.</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="customCssUrl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>customCssUrl</zorder>
   <zorder>customCssEnable</zorder>
   <zorder>customCssSelect</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>uiMenuAndToolbar</tabstop>
  <tabstop>uiMenuOnly</tabstop>
  <tabstop>uiToolbarOnly</tabstop>
  <tabstop>bgColorButton</tabstop>
  <tabstop>customCssEnable</tabstop>
  <tabstop>customCssUrl</tabstop>
  <tabstop>customCssSelect</tabstop>
  <tabstop>statusBarEnable</tabstop>
  <tabstop>enableLoadLast</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bgColorButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>bgColorChoose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customCssSelect</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>customCssSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customCssEnable</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>customCssEnable(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>bgColorChoose()</slot>
  <slot>customCssSelect()</slot>
  <slot>customCssEnable(int)</slot>
 </slots>
</ui>
